### PR TITLE
Docs: add needed key in expo plugin options doc

### PR DESCRIPTION
**Pull request by rlemasquerier**
Good morning !

Thank you for providing this expo plugin (hey @louiszawadzki 😄 - hope you are doing well ! 🙂 )

I noticed this small issue with the doc, I think this extra key is needed to the config options if I'm not missing anything !

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)
- [ ] If this PR is auto-generated, please make sure also to manually update the code related to the change

### CODE REVIEW

**Review by louiszawadzki — APPROVED** [review]
Hi! 

Thanks a lot for catching this and taking time to open a PR @rlemasquerier :) 
Happy to see you're using Datadog, feel free to reach out if you need any help on using the SDK :)

**Comment by louiszawadzki** [comment]
I'll merge this once we get a review from our documentation team :)

**Review by buraizu — APPROVED** [review]

**Pull request merged**
